tools/pagerduty-migrator/scripts/add_users_pagerduty_to_grafana.py

```diff
@@ -0,0 +1,44 @@
+import os
+import secrets
+import sys
+import requests
+
+from urllib.parse import urljoin
+from pdpyras import APISession
+
+PAGERDUTY_API_TOKEN = os.environ["PAGERDUTY_API_TOKEN"]
+PATH_USERS_GRAFANA = "/api/admin/users"
+URL_GRAFANA = os.environ["URL_GRAFANA"] # Example: http://localhost:3000
+USERNAME_GRAFANA = os.environ["USERNAME_GRAFANA"]
+PASSWORD_GRAFANA = os.environ["PASSWORD_GRAFANA"]
+SUCCESS_SIGN = "✅"
+ERROR_SIGN = "❌"
+
+def pd_list_users():
+
+    session = APISession(PAGERDUTY_API_TOKEN)
+
+    users = session.list_all("users", params={"include[]": "notification_rules"})
+
+    for user in users:
+        password = secrets.token_urlsafe(15)
```

Comment:
I'm not entirely sure that a new user will be able to login without knowing the password, do we assume the user will reset their password?

Reply:
I added this kind of information in the README

https://github.com/grafana/oncall/pull/403/commits/1bff154af6c2cad1ba19a1390780f4e3ca0b22be